select_imputation_method/README.md

```diff
@@ -0,0 +1,62 @@
+# Selecting an imputation method for refine.bio species compendia
+
+### Background
+
+[refine.bio](https://github.com/AlexsLemonade/refinebio) contains gene expression data measured with multiple technologies and many platforms. 
+For instance, there are over 25 refine.bio-supported microarray platforms for human assays alone. 
+Microarrays, which currently comprise the bulk of publicly available gene expression data, are limited by the sequences contained on the array itself; 
+older arrays measure fewer genes compared to more recent platforms. 
```

Comment:
If this is the same sentence as the semicolon indicates, you can leave it on the same line right?

Reply:
You can but I think it's easier to review/comment on the text this way with no adverse effect on the rendered markdown.